ext/js/dictionary/dictionary-database.js

```diff
@@ -34,6 +36,8 @@ export class DictionaryDatabase {
         this._createOnlyQuery3 = (item) => IDBKeyRange.only(item.term);
         /** @type {import('dictionary-database').CreateQuery<import('dictionary-database').MediaRequest>} */
         this._createOnlyQuery4 = (item) => IDBKeyRange.only(item.path);
+        /** @type {import('dictionary-database').CreateQuery<import('dictionary-database').DrawMediaGroupedRequest>} */
+        this._createOnlyQuery5 = (item) => IDBKeyRange.only(item.path);
```

Comment:
Allow me to follow this up with another PR which cleans up this code a bit (it can be resolved with better variable names for all of these but it clutters up this PR)